### ISSUE TITLE
Integrate Better AI Empires Difficulty Changes.

### ISSUE THREAD

**Issue by Thom293** [captaincobbs/ESG-Mod#1236]
First - Ive used Git before only to post bugs.  So if I do any of this wrong please let me know and I will correct.

Per request, will integrate the Better AI Empires Difficulty Changes.  This will apply only to AI Empires, and not Empires that are Player-controlled.

https://steamcommunity.com/sharedfiles/filedetails/?id=2602129157

Only the ones in the following files:

SimulationDescriptors[GameDifficulty].xml
SimulationDescriptors[Empire].xml
SimulationDescriptors[FactionTrait].xml (and the DLC versions)

**Comment by stygmaskyzoo** [comment]
remember to work on development1-5-1 branch and not main!

**Comment by Arpharnator** [comment]
you're not gonna wanna fork, otherwise you can't merge

**Comment by Arpharnator** [comment]
also who requested this @captaincobbs

**Comment by Thom293** [comment]
Changes uploaded.  Not quite sure how to do the pull request and dont want to mess anything up.  Will leave that up to yall.

Tested it in 3 normal speed games and 1 fast speed.  Seems to work.  UC won by wonder victory on turn 66 in the fast game.  All of the races were well balanced for my mod - but ESG makes additional changes. 

So, maybe undertuned or overtuned.  I dont know ESG as well as yall.  Maybe will need a tweak.  I would say that Serious & Impossible are the most fair difficulty now.  Endless has a buff that I cannot beat, but Im bad at the game.

If more is wanted, I still need to balance AI ship designs and the Vodyani.  I "fix" both in my mod in a different way that will not work with ESG, so I will have to attack those issues from another route.

These are not all of the changes from my mod, but they are the ones I can do without interfering in MP.  Here are the change notes from my mod for these changes:

CHANGES TO SERIOUS - ENDLESS DIFFICULTY

SERIOUS

All AI races now get appropriate benefit (9-12% research cost reduction) from Science Juggernaughts. They only got the flat buff before because once they make them they do not orbit special nodes.
AI ONLY races have had their overcolonization cap removed. It only hurt the AI.
AI ONLY Hissho turned into a wide snowball race. Given FIDS buff per system owned, to simulate Mining Experts, since they got no benefit from mining probes (I have never seen an AI race use one.)
AI ONLY Sophon, Unfallen, Vaulters, Given Base Industry buff because they will not build industry buildings reliably, or at all.
AI ONLY Nakalim given industry bonus per relic and probe bonus to find them. Nakilim don’t build industry buildings either, and don’t use their empire wide % buffs from Relics.
AI only Vaulters received a buff to Argosy cooldown.
AI ONLY Horatio given small industry buff per population, because has a crappy primary specialty for late game: Food and doesnt appear to use the buildings that would help him. He can get spicy late game if allowed to breed, or, whatever it is he does.

IMPOSSIBLE DIFFICULTY:

All of the above changes, plus:
AI given small manpower per turn bonus.
AI given small deployment bonus.
AI given small happiness bonus.
AI given upkeep reductions.
AI Vodyani given life force bonus.
Hissho given Honor bonus.
AI Races Given a small Quickstart FIDS Bonus

ENDLESS DIFFICULTY:

All of the above changes, plus:
AI Given Significant Quickstart FIDS Bonus and small move bonus.

Endless Difficulty: I have squeezed all I can out of the AI, and I think I did a decent job. But its still no human. So, now Endless gives them a bonus to simulate "human cheese tactics" in game. The AI gets a flat FIDS buff on Endless difficulty now. This is different from all of the other difficulty changes by Amplitude or other mods because it gives them an immediate bonus on each outpost and gives them a much quicker start which means they can be more aggressive quicker. If the current setting is too hard or too easy, Increase or Decrease the number. Right now the AI gets an 50 Flat FIDS bonus, and 1 movement point bonus.
